server/routerlicious/packages/local-server/src/packageVersion.ts

```diff
@@ -6,4 +6,4 @@
  */
 
 export const pkgName = "@fluidframework/server-local-server";
-export const pkgVersion = "0.1036.2000";
+export const pkgVersion = "0.1036.3000";
```

Comment:
Is this correct? You're consuming a pre-release of this version in this change. Don't you have to publish the server prerelease before you can make the changes in the local-driver?

Reply:
Oh, is this because the server change was actually in .2000?